### DISCUSSION TITLE
makeglossaries und automake bei LuaLaTeX

### DISCUSSION THREAD

**Discussion by LauraCharlotteH** [tudace/tuda_latex_templates#391] [General LaTeX Questions] [answered]
Hallo,
erst mal vielen Dank für die Arbeit an diesen Vorlagen/Paketen!

 Dann zur Frage:
Ich versuche für meine Thesis eine Liste der Abkürzungen einzubauen. Dazu verwende ich das glossaries Paket. Ein "minimal working" in einem einfachen Article Dokument funktioniert. 

Aber sobald ich es in der TUDa Thesis Vorlage versuche wird die Liste der Abkürzungen auch bei mehrmaligem rekompilieren nicht angezeigt. Die Abkürzungen im Text funktionieren.

Ich habe inzwischen verschiedenste Versionen und Paktete ausprobiert, aber bisher nichts funktionierendes gefunden.

Angehängt sind das funktionierende minimal Beispiel sowie die main.txt der nicht funktionierenden Variante in der Thesis Vorlage (Inhalt gekürzt).

über Hilfe oder Hinweise würde ich mich sehr freuen!
schönen Gruß,
Laura

[working-minimal.txt](https://github.com/tudace/tuda_latex_templates/files/9275630/working-minimal.txt)
[main.txt](https://github.com/tudace/tuda_latex_templates/files/9275632/main.txt)

**Comment by TeXhackse** [comment, marked as answer]
Das gnaze hat nicht direkt was mit tuda-ci zu tun. Vielmehr damit, dass tuda-ci lualatex empfiehlt, das „working“ ist mit pdflatex gebaut.  Der Unterschied wrd durch den „Magic Comment“ in der `main` erzeugt. 

```
! TeX program=lualatex
```

lualatex verfügt über ein paar mehr Sicherheitsmechanismen, weswegen das automake da nicht ohne weiteres funktioniert.
Man kann das dort hinkriegen, indem man glossaries statt `automake` die Option `automake=immediate` und das Paket `shellesc`.


[main.txt](https://github.com/tudace/tuda_latex_templates/files/9275686/main.txt)

Offtopic Hinweis:
UTF-8 ist auf aktuellen Systemen eh Standard,  daher macht es da keinen Unterschied, aber die Einbindung von `\usepackage[utf8]{inputenc}`, die nur aus Kompatibilitätsgründen enthalten ist, sollte immer die erste bleiben. Bei älteren Versionen wäre sonst TeX sehr darüber verwirrt, dass das Paket mit anderer Eingabekodierung geladen wird. Eigene Pakete gerne drunter einfügen.

**Reply by LauraCharlotteH** [reply]
Super!
Vielen Dank für die schnelle Reaktion und die ganzen zusätzlichen Erklärungen. Wieder was gelernt :)
schöne Woche wünsche ich
